packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -552,8 +552,10 @@ export const Dropdown = (React.forwardRef<HTMLDivElement, DropdownProps>((props,
       },
       'aria-invalid': ariaInvalid,
       'aria-label': undefined,
-      'aria-labelledby': [ariaLabelledby, triggerButtonId].filter(l => !!l).join(' '),
-      ...(open && { 'aria-expanded': true }),
+      ...(ariaLabelledby && {
+        'aria-labelledby': [ariaLabelledby].filter(l => !!l).join(' '),
+        ...(open && { 'aria-expanded': true }),
```

Comment:
Before the change, `aria-expanded` is applied regardless of the `ariaLabelledby` value.
With this change, `aria-expanded` won't be applied unless `ariaLabelledby` is defined. Is this intentional?

Reply:
Updated. I wonder if we should add a unit test for that.